web/components/Feed/Feed.tsx

```diff
@@ -311,6 +313,11 @@ function Feed(props: {
     getInitialFilters()
   );
 
+  const allPosts = trpc.post.getAllPosts.useQuery().data;
```

Comment:
It seems like you're missing some code from our current version of `staging` - make sure to update your local `staging` branch to the remote's

Reply:
All of the database persistence logic was handled by #109, be sure you have those changes